### PR TITLE
{lang}[GCCcore/8.2.0] Ruby v2.6.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3129.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6b692086ca65008447035187493cbdc1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d87f96ceacb03408babf7a9edd27622c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2418.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/70537ee482f565baaeb6729082529c25 for a full test report.

File: easybuild/easyconfigs/r/Ruby/Ruby-2.6.3-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+version = '2.6.3'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://cache.ruby-lang.org/pub/ruby/%(version_major_minor)s']
```
Comment (akesandgren) [inline easybuild/easyconfigs/r/Ruby/Ruby-2.6.3-GCCcore-8.2.0.eb]: https ?? or doesn't it work on that one?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/1eed966898b726e6bf449794c5800f6b for a full test report.

File: easybuild/easyconfigs/r/Ruby/Ruby-2.6.3-GCCcore-8.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/r/Ruby/Ruby-2.6.3-GCCcore-8.2.0.eb]: `https` does work, fixed in dbf3c21e5

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**